tests/manage/mcg/test_multicloud.py

```diff
@@ -0,0 +1,77 @@
+import logging
+
+import pytest
+
+from ocs_ci.framework.pytest_customization.marks import tier1
+from ocs_ci.framework.testlib import MCGTest
+
+logger = logging.getLogger(__name__)
+
+
+class TestMultiRegion(MCGTest):
```

Comment:
Should this class be called `TestMultiCloud`? We already have such class in tests/manage/mcg/test_multi_region.py so I wasn't sure

Reply:
Whoops. You're right. That's what I get for copy-pasting entire files.
Thanks :)